src/components/App/ExpressRecorder.tsx

```diff
@@ -733,24 +718,43 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                 <div
                     className={`express-recorder__controls ${styles["express-recorder__controls"]}`}
                 >
-                    {!doRecording && !doCountdown && !doPlayback && (
-                        <button
-                            className={`xr_controls__start ${styles["controls__start"]}`}
-                            id="startRecord"
-                            onClick={this.handleStartClick}
-                            aria-label={this.translator.translate(
-                                "Start Recording. recording will start in a three seconds delay"
-                            )}
-                            tabIndex={0}
-                        />
-                    )}
                     {doRecording && (
                         <RecordingTimer
                             onButtonClick={this.handleStopClick}
                             maxRecordingTime={maxRecordingTime}
                             setStopButtonRef={this.setStopButtonRef}
                         />
                     )}
+                    <div className={styles["settings-wrap"]}>
+                        {!doPlayback && !doRecording && !doCountdown && (
```

Comment:
I get it. you use to cut and paste the logic from one location to another. but is it hard to follow if the condition was met.
therefore, I suggest wrapping a variable that holds the answer to show the `<Settings />` or not.

please create a variable that holds your logic (I believe you should call him "isPlaying") and wrap it with **useMemo** hook

```
const isPlaying = useMemo(() => {
  if(isPlayback){
    return true;
  }
  if(){
    return ...
  }
  return false;
}, [isPlayback, ...]);

```

and after that, you can use:
```
<div className={styles["settings-wrap"]}>
{ !isPlaying ? <Settings /> : null }
</div>
```
once again. do not use `&&` for conditional rendering.

Reply:
adding state variable for showing settings require managing 1 more state.. while I can conclude it from the others.. so I'm not sure it's better.

useMemo does not exist in preact. they recently added something similar and other hooks. I agree it's nice to have but it's require time - I would add hooks all over. @nireliyahu what do you think?